clouds/bigquery/modules/doc/quadbin/QUADBIN_DISTANCE.md

```diff
@@ -0,0 +1,23 @@
+## QUADBIN_DISTANCE
+
+```sql:signature
+QUADBIN_DISTANCE(origin, destination)
+```
+
+**Description**
+
+Returns the **Chebyshev distance** between two quadbin indexes. Returns `null` on invalid inputs.
```

Comment:
perhaps "the **Chebyshev distance** in quadbin cells between two quadbin indexes" ? but I'm not sure, it might be confusing... the **Chebyshev distance** in number of quadbin cells between two quadbin indexes" ?? I'm not sure yet.

We could also add a comment about both indices needing to be of the same resultion.

Reply:
On second thought, if the reader understands what Chebyshev distance is (or they have looked it up), they probably have figured out what this that, so we can leave that sentence as it. But I would add:
> The origin and destination indices must have the same resolution. Otherwise NULL will be returned.

Reply:
I'll also had the link to wikipedia as we do in here:
https://github.com/CartoDB/analytics-toolbox-core/blob/bug/sc-318681/missing-quadbin-distance-in-at/clouds/redshift/modules/doc/quadbin/QUADBIN_KRING_DISTANCES.md#L36